### DISCUSSION TITLE
[Question]: Is there a way to change Default page?

### DISCUSSION THREAD

**Discussion by tip-dteller** [danny-avila/LibreChat#3034] [Q&A]
### What is your question?

We have a Librechat instance configured for our team with Allowed Assistants and a single model to use.

Suppose a user worked with the Assistants and finished their work, closed the browser tab and later opened Librechat again, it puts the user back with Assistant. (Last Used Assistant\Endpoint)

Is there a way to force a specific default?

meaning that whenever users login\open a librechat tab it will always go to a specific endpoint instead of Last Used.

Why does it matter?
because if a user reopens the tab\page and questions Assistant A which is instructed for Learning but the user actually needs Assistant B which is instructed for Code.

This is to minimize incorrect endpoint usage and potentially waste tokens, since users may neglect switching to what they really need. 

### More Details

no

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Are they logging out or sharing accounts? If the latter, this is expected behavior. If the account is shared, you can add a default preset to override this behavior.

**Reply by d-teller** [reply]
What do you mean sharing accounts?

Each user logs in with MS Entra, since the login is retained for 2 weeks until prompted to login again.

**Reply by danny-avila** [reply]
I see. Sorry for the confusion, I assumed you meant in a shared browser scenario. 

In any case the default behavior is to use last endpoint/model/assistant. it is mostly possible to circumvent this on a user-level. To make a "default preset", in order to control which endpoint is selected on login, every time, you would need to make use of [modelSpecs](https://www.librechat.ai/docs/configuration/librechat_yaml/object_structure/model_specs)
